docs/content/dagster-cloud/getting-started.mdx

```diff
@@ -1,35 +1,341 @@
+   - [Docker](/dagster-cloud/deployment/agents/docker)
+   - [Kubernetes](/dagster-cloud/deployment/agents/kubernetes/configuring-running-kubernetes-agent)
+   - [Local](/dagster-cloud/deployment/agents/local)
+
+Once Dagster Cloud receives an agent heartbeat, click **Continue** to move on to the next step.
+
+**Step 3.2: Add a Dagster Cloud code location**
+
+For most Hybrid deployments - with the exception of those using a local agent - you'll need to create a Docker image containing your Dagster code and then add a code location to Dagster Cloud that references the image.
+
+1. Click **Configure a Code Location**.
+
+2. In the **Code Locations** tab that displays, click **+ Add**.
+
+3. A pre-filled YAML editor with configuration will display.
+
+4. Verify that the configuration is correct, then click **Add code location** to save.
+
+The agent will attempt to load your code. After the code is loaded, the new code location will show a **Loaded** status and a sample Dagster job will display in Dagster Cloud.
+
+When finished, [continue to the next step](#step-4-set-up-cicd).
+
+</TabItem>
+</TabGroup>
+
+---
+
+## Step 4: Set up CI/CD
+
+<Note>
+  If you're using Serverless deployment <strong>and</strong> you used one of our
+  templates to deploy your code, this step has already been completed for you!
+  CI/CD comes pre-configured in our starter projects.
+</Note>
+
+In this step, you'll set up Continuous Integration (CI) and Continuous Deployment (CD). Completing this step will enable [Branch Deployments](/dagster-cloud/developing-testing/branch-deployments) and continuous code location deployments for your Dagster project.
+
+<TabGroup>
+<TabItem name="GitHub Actions">
+
+For continunous integration using GitHub actions:
+
+1. Copy the workflow files (`.github/workflows`) from the quickstart repository for your deployment type:
+
+   - [Serverless](https://github.com/dagster-io/dagster-cloud-serverless-quickstart/tree/main/.github/workflows)
+   - [Hybrid](https://github.com/dagster-io/dagster-cloud-hybrid-quickstart/tree/main/.github/workflows)
+
+2. **For Hybrid deployments**: In both workflow files, uncomment the step that is relevant to your Docker container registry. For example, if using DockerHub, you'd uncomment the `DockerHub` step in these files.
+
+3. Set the necessary GitHub action secrets, including your Dagster Cloud API Token and Organization ID. Refer to the [GitHub docs](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository) for more info about GitHub Action secrets.
+
+</TabItem>
+<TabItem name="Other CI/CD provider">
+
+<Note>This is applicable only to Hybrid deployments.</Note>
+
+For continuous integration using a CI/CD provider other than GitHub, your system should:
+
+1. Build a Docker image containing your Dagster code.
+2. Push the image to a registry your agent can access.
```

Comment:
Little stuck here. I know Serverless *can* deploy using the CLI (`dagster-cloud serverless deploy`) but I'm not sure what needs to change here to make this applicable to Serverless.